### PR TITLE
pipelines list

### PR DESCRIPTION
**Pull request by maheshp**

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb [hunk 1]
```diff
@@ -22,6 +22,29 @@ class PipelinesController < ApiV1::BaseController
       before_action :check_if_pipeline_by_same_name_already_exists, :check_group_not_blank, only: [:create]
       before_action :check_for_stale_request, :check_for_attempted_pipeline_rename, only: [:update]
 
+      def index
+        render :json => pipelines_json
+      end
+
+      def pipelines_json
+        pipelines = pipeline_config_service.pipelinesFor(current_user)
+        pipelines.collect() do |pipeline|
```
Comment (ketan) [inline server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb]: use `inject`?

File: server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb [hunk 2]
```diff
@@ -22,6 +22,29 @@ class PipelinesController < ApiV1::BaseController
...
+          hash = {}
+          hash[:name] = pipeline.name
+          hash[:stages] = stages_json(pipeline.stages)
+          hash
+        end
+      end
+
+      def stages_json stages
+        stages.collect() do |stage|
```
Comment (ketan) [inline server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb]: use `inject`?

File: common/src/com/thoughtworks/go/presentation/Stage.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    public String getName() {
+        return name;
+    }
+
+    public List<Job> getJobs() {
+        return jobs;
+    }
+
+    private List<Job> jobs;
```
Comment (ketan) [inline common/src/com/thoughtworks/go/presentation/Stage.java]: Move this to top?

File: common/src/com/thoughtworks/go/presentation/Pipeline.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    public String getName() {
+        return name;
+    }
+
+    public List<Stage> getStages() {
+        return stages;
+    }
+
+    private String name;
+    private List<Stage> stages;
```
Comment (ketan) [inline common/src/com/thoughtworks/go/presentation/Pipeline.java]: Move these vars to top?

**Comment by maheshp** [comment]
@ketan I must have sent a PR by mistake this is still WIP rather was just spiking out some stuff.

**Pull request closed without merging**